src/jormungandr/testing/jormungandr-integration-tests/src/jormungandr/genesis/pool_update.rs

```diff
@@ -18,7 +18,9 @@ pub fn update_pool_fees_is_not_allowed() {
     let (jormungandr, stake_pools) = startup::start_stake_pool(
         &[stake_pool_owner.clone()],
         &[],
-        Block0ConfigurationBuilder::default(),
+        Block0ConfigurationBuilder::default()
+            .with_slots_per_epoch(20.try_into().unwrap())
```

Comment:
As you did above, should we consider creating variables for `20` and `2` too?

Reply:
I didn't add a variable here because there is only one test in the file